desktop/server/server.js

```diff
@@ -1,59 +1,62 @@
-'use strict';
-
 /**
  * External Dependencies
  */
-const portscanner = require( 'portscanner' );
-const log = require( 'lib/logger' )( 'desktop:server' );
+import { dialog } from 'electron';
+import http from 'http';
+import portscanner from 'portscanner';
+import logFactory from 'lib/logger';
```

Comment:
Looks like this should be moved to the internal dependencies section.

Reply:
Done! :+1: